#!/usr/bin/env python
PACKAGE = "aiv_controller_pid"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()


gen.add("x_kp", double_t, 0, "kp value x", 1, 0, 1000)
gen.add("x_ki", double_t, 0, "ki value x", 1, 0, 1000)
gen.add("x_kd", double_t, 0, "kd value x", 1, 0, 1000)
gen.add("y_kp", double_t, 0, "kp value y", 1, 0, 1000)
gen.add("y_ki", double_t, 0, "ki value y", 1, 0, 1000)
gen.add("y_kd", double_t, 0, "kd value y", 1, 0, 1000)
gen.add("th_kp", double_t, 0, "kp value th", 1, 0, 1000)
gen.add("th_ki", double_t, 0, "ki value th", 1, 0, 1000)
gen.add("th_kd", double_t, 0, "kd value th", 1, 0, 1000)

exit(gen.generate(PACKAGE, "aiv_controller_pid", "controllerServer"))
